tests/tests.cairo

```diff
@@ -1,139 +1,154 @@
-use raito::validation::bits_to_target;
-use raito::validation::target_to_bits;
+use super::blocks::{block_0::block_0, block_170::block_170};
+use raito::state::{ChainState, UtreexoState};
+use raito::validation::{BlockValidatorImpl};
+
+#[test]
+fn test_block0() {
+    let block = block_0();
+    let prev_chain_state = ChainState {
```

Comment:
Maybe it makes sense to turn it into a helper in raito? Initial chain state won't change anytime